### PR TITLE
Update cmake

### PR DESCRIPTION
**Pull request by dj2**

### CODE REVIEW

File: kokoro/scripts/linux/build.sh [hunk 1]
```diff
@@ -53,12 +53,12 @@ fi
 echo y | sudo apt-get purge --auto-remove cmake
 
 # Installing the 3.10.2 version
```
Comment (ben-clayton) [inline kokoro/scripts/linux/build.sh:55]: Is this comment still correct?

Comment (dj2) [reply kokoro/scripts/linux/build.sh]: nope, removed

**Comment by dj2** [comment]
PTAL. Updating this as Swiftshader is moving to a later CMake so want to make sure our bots are ready for the change.

File: kokoro/scripts/linux/build_dawn.sh [hunk 2]
```diff
@@ -43,12 +43,13 @@ fi
 echo y | sudo apt-get purge --auto-remove cmake
 
 # Installing the 3.10.2 version
```
Comment (alan-baker) [inline kokoro/scripts/linux/build_dawn.sh:45]: Bad comment now?

Comment (dj2) [reply kokoro/scripts/linux/build_dawn.sh]: done

**Pull request merged**